src/views/Auctions/components/AuctionNavBar/index.spec.tsx

```diff
@@ -1,7 +1,7 @@
 // Externals
 
 import React  from 'react'
-import { render, cleanup } from '@testing-library/react'
+import { render, cleanup, fireEvent, waitFor } from '@testing-library/react'
```

Comment:
You did not use them. Please check lint issues again. And fix all those lint errors